app/services/build_new_actor.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class BuildNewActor
+  class << self
+    def call(psu_id: nil, orcid: nil)
+      if psu_id.present?
+        build_with_psu_id(psu_id)
+      elsif orcid.present?
+        build_with_orcid(orcid)
+      else
```

Comment:
There's another case I missed here: what if _both_ are present. The Penn State id would take precedence, but I'm not sure we would want that.